tests/paths.js

```diff
@@ -0,0 +1,11 @@
+import path from 'path';
```

Comment:
At top of this file (above the line above), please add a JSDoc-style comment saying what this file does.